### DISCUSSION TITLE
shift a bunch of lines

### DISCUSSION THREAD

**Discussion by neomaxmaya** [SubtitleEdit/subtitleedit#5985] [Q&A]
Hello, 
I have merged a few subtitles together.
How do i shift some of them to allow for a silent gap in between?
Thanks

**Comment by devkatalist** [comment]
Hello, @neomaxmaya. Go to Synchronisation > Adjust all times. This option will allow to shift All lines, Selected lines or Selected and Subsequent lines. Use Show earlier to move them back and Show later to move them forward. Hope this helps.
